utils/post_utils.jsx

```diff
@@ -123,3 +123,21 @@ export function containsAtChannel(text) {
 
     return (/\B@(all|channel)\b/i).test(mentionableText);
 }
+
+export function getOldestPostIdFromPosts(posts) {
```

Comment:
These could also be moved to mattermost-redux for use in the mobile app as well

Reply:
Yes moving it.